packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -89,7 +91,7 @@ class PinCodeBloc extends $PinCodeBloc {
 
   @override
   ConnectableStream<dynamic> _mapToAuthenticatedState() => Rx.merge([
-        _digitsCountState.switchMap((digitsCount) =>
+        digitsCount.flatMap((digitsCount) =>
```

Comment:
Use the where operator to ensure the authentication triggers at the right moment

Reply:
Done